### ISSUE TITLE
:wheelchair: a11y(bal-hint, bal-field-hint): hints are not accessible via Keyboard & Screenreader

### ISSUE THREAD

**Issue by zsofia-oroszi-ibmix** [baloise/design-system#1629]
### Description of this issue

The info icon is not accessible with keyboard and screenreader => opening the popup is not possible via keyboard

### How to fix it

The icon should be accessible via Tab and function as a button (opening the popup with Enter/Space).

### Code Reproduction URL

https://design.baloise.dev/?path=/story/components-data-display-hint--field-hint&globals=framework:html

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by GinaBiondo** [comment]
- The hint is focusable via keyboard navigation and the hint opens correctly. However the focus state is of such subtle nature, that it is barely noticeable. An outline as implemented for the buttons or similar would be preferred.

- Also opening the hint via spacebar scrolls the page to the bottom. This does not happen, when enter key is used to open the hint. Both keys should work similar without scrolling the page
